src/staging/src/pages/index.js

```diff
@@ -67,243 +57,41 @@ export const query = graphql`
-                  <a href="/conserve" className="card grid-card grid-card--horz">
-                    <div className="card-img">
-                      <img src={imgConservation} alt="A mountain peak" />    
-                    </div>
-                    <div className="card-body">
-                      <h5 className="card-body-header">
-                        Conservation
-                      </h5>
-                      <div className="card-button">
-                      Learn more <i className="fa fa-chevron-circle-right"></i>
-                      </div>
-                    </div>
-                  </a>
-                </div>
-                <div className="col-12 home-card-wrapper">
-                  <a href="/news" className="card grid-card grid-card--horz">
-                    <div className="card-img">
-                      <img src={imgGetToKnow} alt="Family walking on a trail" />    
-                    </div>
-                    <div className="card-body">
-                      <h5 className="card-body-header">
-                        Get to know BC Parks
-                      </h5>
-                      <div className="card-button">
-                      Learn more <i className="fa fa-chevron-circle-right"></i>
-                      </div>
-                    </div>
-                  </a>
-                </div>
-                <div className="col-12 home-card-wrapper">
-                  <a href="/get-involved" className="card grid-card grid-card--horz">
-                    <div className="card-img">
-                      <img src={imgGetInvolved} alt="People holding license plates" />    
-                    </div>
-                    <div className="card-body">
-                      <h5 className="card-body-header">
-                        Get Involved
-                      </h5>
-                      <div className="card-button">
-                      Learn more <i className="fa fa-chevron-circle-right"></i>
-                      </div>
-                    </div>
-                  </a>
-                </div>
-              </div>
+          <div className="home-page-search-bg">
+            <StaticImage src="../images/home/search_bg.png"
+              placeholder="blurred"
+              loading="eager"
+              style={{ display: "block" }}
+              alt="Mount Robson Park" />
+          </div> 
+        </div>
+      </div>
+      <div className="home-content-width-override">
+        <div id="main">
+          {pageContent.map(content =>
+            <div key={content.strapi_component + '-' + content.id}>
+              <PageContent contentType={content.strapi_component} content={content}></PageContent>
             </div>
-          </Container>
-        </>
-      )}
-      <Container className="max-width-override" fixed disableGutters>
+          )}
+        </div>
+      </div>
+      <div className="home-max-width-override" fixed disableGutters>
```

Comment:
Can remove `fixed disableGutters` here.  They were here when the `div` was a `Container` component from Material UI.  They don't do anything now and are creating errors in the console

Reply:
yep, thx.